src/app/docs/tabs/tabs-docs.component.html

```diff
@@ -4,10 +4,373 @@
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/tabs/tabs-behavior-1.png"
+        >
+        </sky-docs-design-guideline-thumbnail>    
+        </sky-docs-design-guideline>
+        
+        <sky-docs-design-guideline
+        heading="States"
+      >
+        <table class="sky-docs-table">
+          <tr>
+            <td class="sky-docs-table-cell sky-docs-table-cell-primary">
+              Selected
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/tabs/tabs-state-selected.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Unselected
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/tabs/tabs-state-unselected.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Hover
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/tabs/tabs-state-hover.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Collapsed, closed
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/tabs/tabs-state-collapsed-closed.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              Collapsed, open
+            </td>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+                imageSource="~/assets/img/guidelines/tabs/tabs-state-collapsed-open.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+          </tr>
+        </table>
+      </sky-docs-design-guideline>
+        
+    </sky-docs-demo-page-section>
+      
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+    
+        <div class="sky-text-block">
+            If tabs are the main navigation on a page, display the tab header in the page title. Use the following format: <b>&lt;Page title&gt; — &lt;Company, product, or site identifier&gt;</b>. For example: <b>Portfolio — Work center — Blackbaud</b>. In addition, follow these guidelines for page titles:
```

Comment:
I'd change the second sentence to: `use the following format for tab titles: *<Tab title> — <Page title> — <Company, product, or site identifier>*.`

And what do you think about tightening up by eliminating the bullet list here? We could pull the first two bullets into the third sentence and then add a cross reference: `In addition, follow guidance for page titles such as keeping titles short and making them unique. For information about how to specify titles, see the page title guidelines." (With "the page title guidelines" linking to https://developer.blackbaud.com/skyux/design/guidelines/page-layouts.

Reply:
I like this. It also allows us to trim the Angular reference material since it's already present on those page design guidelines.